tests/test_standard.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2017-2019 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02D111-1307, USA.
+"""Book fields tests."""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from cds_dojson.marc21.models.books.standard import model
+from cds_dojson.marc21.utils import create_record
+
+marcxml = ("""<collection xmlns="http://www.loc.gov/MARC21/slim">"""
+           """<record>{0}</record></collection>""")
+
+
+def check_transformation(marcxml_body, json_body):
+    blob = create_record(marcxml.format(marcxml_body))
+    record = model.do(blob, ignore_missing=False)
+    expected = {
+        '$schema': {
+            '$ref': 'records/books/book/book-v.0.0.1.json'
+        }
+    }
+    expected.update(**json_body)
+    assert record == expected
+
+
+def test_titles(app):
+    with app.app_context():
+        check_transformation(
+            """
+            <datafield tag="245" ind1=" " ind2=" ">
```

Comment:
@agentilb is it possible  / OK for a record to have two 245 fields? in MARC 245 is non-repeatable. It will be quite difficult to choose what title to display as well. WDYT?

Reply:
If this is the case, we can fix all records with multiple occurrences of 245 before migrating

Reply:
Normally, in MARC we have only 1 245, but we can have several 246. 
Can you send me the list of cases where we have several 245?